releases/release-1.26/release-notes/maps/pr-108501-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 108501
+releasenote:
+  text: Enabled `kube-controller-manager` to support '--concurrent-horizontal-pod-autoscaler-syncs'
```

Please use backticks when a flag or command is found:

```suggestion
  text: Enabled `kube-controller-manager` to support `--concurrent-horizontal-pod-autoscaler-syncs`
```